src/includes_normalize-win32.cc

```diff
@@ -53,16 +53,41 @@ bool SameDriveFast(StringPiece a, StringPiece b) {
   return IsPathSeparator(a[2]) && IsPathSeparator(b[2]);
 }
 
+void GetFullPathNameSafeLongPath(const string &filename, size_t absoluteSize, char *absolute) {
+  // NOTE: Avoiding malloc for speed
+  wchar_t wfilename[_MAX_PATH + 1];
+  {
+    size_t charsToConvert = filename.size() + 1;
+    size_t convertedChars = 0;
+    mbstowcs_s(&convertedChars, wfilename, charsToConvert, filename.c_str(), sizeof(wfilename) / sizeof(wfilename)[0]);
```

Comment:
We don't currently do any multibyte conversion anywhere -- this seems orthogonal to this patch (?)

(also, 80 cols)